### PR TITLE
made minor fixes

### PR DESCRIPTION
**Pull request by pablo-clueless**

### CODE REVIEW

File: src/sections/APIHubTab.tsx [hunk 1]
```diff
@@ -87,25 +87,24 @@ const APIHubTab: React.FC = () => {
...
-    general: <Lightbulb />,
-    sports: <SportsFootball />,
-    travel: <AirplanemodeActive />,
-    finance: <AttachMoney />,
-    educational: <School />,
-    data: <DataArray />,
-    science: <Science />,
-    music: <MusicNote />,
-    tools: <Build />,
+    "popular apis": <LibraryBooks />,
```
Comment (PhilemonBrain) [inline src/sections/APIHubTab.tsx]: please revert this.. it'll break on prod

Comment (PhilemonBrain) [reply src/sections/APIHubTab.tsx]: and update please

**Review by PhilemonBrain — CHANGES_REQUESTED** [review]
revert the category name mapping. it'll break on prod

File: src/sections/APIHubTab.tsx (hunk 1 above)
Comment (pablo-clueless) [reply src/sections/APIHubTab.tsx]: > please revert this.. it'll break on prod

What will break on prod??

**Comment by pablo-clueless** [comment]
> revert the category name mapping. it'll break on prod

Some of the names are not correct and their icons won't show. If the build is successful, will it still break in prod??

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**